docs/sources/operations/autoscaling_queriers.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Autoscaling Loki queriers
+menuTitle: Autoscaling queriers
+description: Kubernetes deployments of a microservices mode Loki cluster can use KEDA to autoscale the quantity of queriers.
+weight: 30
+---
+
+# Autoscaling Loki queriers
+
+A microservices deployment of a Loki cluster that runs on Kubernetes will typically handle a
+workload that varies throughout the day.
+To make Loki easier to operate and optimize the cost of running Loki at scale,
+we have designed a set of resources to help you autoscale your Loki queriers.
+
+## Prerequisites
+
+You need to run Loki in Kubernetes as a set of microservices. You need to use the query-scheduler.
+
+We recommend using [Kubernetes Event-Driven Autoscaling (KEDA)](https://keda.sh/) to configure autoscaling
+based on Prometheus metrics. Refer to the [Deploying KEDA](https://keda.sh/docs/latest/deploy) section to learn more
+about setting up KEDA in your Kubernetes cluster.
+
+## Scaling metric
+
+Queriers pull queries from the query-scheduler queue and process them on the querier workers. Therefore, it makes sense to scale based on:
+
+- The scheduler queue size.
+- The queries running in the queriers.
+
+The query-scheduler exposes the `cortex_query_scheduler_inflight_requests` metric.
+It tracks the sum of queued queries plus the number of queries currently running in the querier workers.
+The following query is useful to scale queriers based on the inflight requests.
+
+```promql
+sum(
+  max_over_time(
+    cortex_query_scheduler_inflight_requests{namespace="loki-cluster", quantile="<Q>"}[<R>]
+  )
+)
+```
+
+We can use the quantile (Q) and the range (R) parameters to fine-tune the metric.
+The higher Q is, the more sensitive the metric is to short-lasting spikes.
+As R increases, we reduce the variation over time in the metric.
```

Comment:
I'm confused about 'we' in this sentence. Does the system automatically reduce variation?

Reply:
No. The user can modify Q and R to fine-tune the metric.